### PR TITLE
updated for CakePHP 3.6

### PR DESCRIPTION
**Pull request by mirko-pagliai**
Please, release a version or create a branch compatible with CakePHP 3.6.

### CODE REVIEW

**Comment by mirko-pagliai** [comment]
Needs phpunit 6.
See https://github.com/cakephp/cakephp/issues/11732

File: composer.json [hunk 1]
```diff
@@ -31,10 +31,10 @@
   },
   "require": {
     "php": ">=5.6",
-    "cakephp/cakephp": "~3.2"
+    "cakephp/cakephp": "~3.6"
```
Comment (ADmad) [inline composer.json]: Bumping to 3.6 is not necessary. The updated methods exist in 3.5 too so please change this to `^3.5`.

File: composer.json [hunk 2]
```diff
@@ -31,10 +31,10 @@
...
   },
   "require-dev": {
-    "phpunit/phpunit": "*",
+    "phpunit/phpunit": "^5.7|^6.0",
```
Comment (ADmad) [inline composer.json]: Ideally `^5.7.14|^6.0` is needed.

File: composer.json [hunk 3]
```diff
@@ -31,10 +31,10 @@
...
     "cakephp/cakephp-codesniffer": "2.*"
```
Comment (ADmad) [inline composer.json]: Can you please update this too to `^3.0`.

**Review by ADmad — CHANGES_REQUESTED** [review]

**Review by ADmad — APPROVED** [review]

**Pull request merged**